services-custom/s3-transfer-manager/src/it/java/software/amazon/awssdk/transfer/s3/S3TransferManagerUploadIntegrationTest.java

```diff
@@ -92,25 +102,26 @@ void upload_file_SentCorrectly() throws IOException {
 
     private static void assertListenerForSuccessfulTransferComplete(CaptureTransferListener transferListener) {
         assertThat(transferListener.isTransferInitiated()).isTrue();
-        assertThat(transferListener.isTransferInitiated()).isTrue();
+        assertThat(transferListener.isTransferComplete()).isTrue();
         assertThat(transferListener.getRatioTransferredList()).isNotEmpty();
         assertThat(transferListener.getRatioTransferredList().contains(0.0));
         assertThat(transferListener.getRatioTransferredList().contains(100.0));
         assertThat(transferListener.getExceptionCaught()).isNull();
     }
 
-    @Test
-    void upload_asyncRequestBody_SentCorrectly() throws IOException {
+    @ParameterizedTest
+    @MethodSource("transferManagers")
+    void upload_asyncRequestBody_SentCorrectly(S3TransferManager transferManager) throws IOException {
```

Comment:
Can we rename this method to `upload_asyncRequestBodyFromString_SentCorrectly` to be consistent with the next?

Reply:
Done